LayoutTests/fast/text/canvas-color-fonts/stroke-gradient-COLR-expected.html

```diff
@@ -4,5 +4,22 @@
 <meta charset="utf-8">
 </head>
 <body>
+<canvas id="canvas" width="700" height="500" style="width: 700px; height: 500px;"></canvas>
+<script>
+let canvas = document.getElementById("canvas");
+let context = canvas.getContext("2d");
+context.lineWidth = 2;
+let gradient = context.createLinearGradient(100, 200, 400, 200);
+gradient.addColorStop(0, "rgb(255, 0, 128)");
+gradient.addColorStop(1, "rgb(255, 153, 51)");
+context.strokeStyle = gradient;
+context.fillStyle = gradient;
+context.fillRect(200, 20, 100, 100);
+context.strokeStyle = "white";
+context.strokeRect(100, 20, 100, 100);
+context.strokeRect(200, 20, 100, 100);
+context.strokeRect(220, 20, 20, 100);
+context.strokeRect(300, 20, 100, 100);
```

Comment:
I am not sure what we expect from the test here. The test draws "BAB" and then cleans the two 'B's leaving only the 'A'. Then it cleans the antialiased pixels in the middle of the 'A'.

So why do not we draw the 'A' only and maybe add pixel tolerance if it is needed?

Reply:
When the test strokes the white "B"s it cuts the edges of "A". Also, I wanted to replicate the test the closest as possible to the steps it takes when drawing.